### DISCUSSION TITLE
SMTP  Mail I forgot my password ERROR

### DISCUSSION THREAD

**Discussion by bleonaraya** [kimai/kimai#2257] [General]
Hello guys, good afternoon,

I have configured the smtp of my .env but still, when I try to recover the password. nothing happens, where can I validate errors?

this is my configuration:
Personal Gmail:

MAILER_URL = smtp: //smtp.gmail.com: 465? Encryption=tls&username=blXXXX@gmail.com&password=BXXXXXXXXXXXX8
MAILER_FROM=bluXXXX@gmail.com

**Comment by kevinpapst** [comment]
The email will not be triggered in a certain timespan, to prevent abuse.
I added an explanation and workaround to the documentation: https://www.kimai.org/documentation/emails.html#troubleshooting
